src/libs/actions/IOU.js

```diff
@@ -1007,6 +1058,12 @@ function getPayMoneyRequestParams(chatReport, iouReport, recipient, paymentMetho
         iouReport.reportID,
         true,
     );
+    const optimisticPersonalDetailsListAction = {
+        accountID: Number(recipient.accountID),
+        avatar: UserUtils.getDefaultAvatarURL(Number(recipient.accountID)),
+        displayName: recipient.displayName || recipient.login,
+        login: recipient.login,
+    };
```

Comment:
There was a copy/paste mistake here. This data itself (without accountID key) is added in `ONYXKEYS.PERSONAL_DETAILS_LIST`. Luckily, this didn't cause any issue but just console error.
More details: https://github.com/Expensify/App/issues/23645#issuecomment-1670255072